streamer/pulp/streamer/server.py

```diff
@@ -84,14 +85,18 @@ class Streamer(resource.Resource):
     """
     Define the web resource that streams content from the upstream repository
     to the client.
-
-    :ivar config: The configuration for this streamer instance.
-    :type config: ConfigParser.SafeConfigParser
```

Comment:
It was moved down to the ``__init__`` block since mhrivnak wanted that doc-blocked.